src/components/Form/DateTimeWidget.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormControl, OutlinedInput } from '@mui/material';
+import { WidgetProps } from '@rjsf/utils';
+
+export const DateTimeWidget = ({
+	id,
+	label,
+	value,
+	disabled,
+	name,
+	readonly,
+	autoFocus,
+	placeholder,
+	onBlur,
+	onChange,
+	onFocus,
+	hideLabel,
+	multiple,
+	options,
+	required,
+}: WidgetProps) => {
+	const change = ({ target: { value } }: any) => {
+		return onChange(value === '' ? options.emptyValue : value);
+	};
+	return (
+		<FormControl variant="outlined">
+			<label htmlFor={id}>
+				{label}
+				{required ? '*' : null}
+			</label>
```

Comment:
Can we please use this: https://mui.com/material-ui/api/input-label/#InputLabel-prop-required

Reply:
Not sure why I left that version, thank you for pointing it out

Reply:
> Our label will not make the MUI animation

It doesn't need to though? It just needs to be displayed 🤔 Can't we just disable the animation and still use InputLabel?

Reply:
the label style is completely different, we would need to create a styled component to make the style match what we need, it is not worth. The label without animation is still not what we want check image 
<img width="261" alt="Screenshot 2023-11-02 at 12 48 34" src="https://github.com/balena-io-modules/ui-shared-components/assets/7238159/0956e519-246e-469e-830c-974ab120b930">

Reply:
@JSReds I don't think that's too much work tbh, here's what I did in the theme to make it look ok:

```ts
		MuiInputLabel: {
			defaultProps: {
				shrink: true,
			},
			styleOverrides: {
				root: {
					transform: 'none',
					position: 'relative',
				},
			},
		},
```

The `sx` as well as the `shrink` props could be set in the theme so we wouldn't need to worry

Reply:
I can update that in a commit if you want?